test/integration/templates/generation scripts/generate.js

```diff
@@ -0,0 +1,14 @@
+const generateHtml = require('./utils/generateHtml');
+const generateJavaSpring = require('./utils/generateJavaSpring');
+const pathToExpectedTemplates = './test/integration/templates/expected/html';
+async function generate() {
+  try {
+    await generateHtml(pathToExpectedTemplates);
+    await generateJavaSpring(pathToExpectedTemplates);
+    // ... more templates to generate
+    console.log('Done generating html templates');
+  } catch (e) {
+    console.log(e);
+  }
+}
+generate();
```

Comment:
This can throw a `UnhandledPromiseRejectionWarning`.
If you wrap it like this:
```
(async () => {
try{
    await generate();
}
catch(error){
  // Some elegant handling
}
})();
```
You'll avoid Node's warning en prevent failures when Node no longer allows uncaught promise rejections.

Reply:
Good catch! Gonna wrap them as you as you suggested with a `handlePromiseGracefully` function :+1: